mfa/oath/src/main/kotlin/com/pingidentity/mfa/oath/OathCredential.kt

```diff
@@ -69,9 +69,9 @@ data class OathCredential(
     val imageURL: String? = null,
     val backgroundColor: String? = null,
     val policies: String? = null,
-    val lockingPolicy: String? = null,
+    var lockingPolicy: String? = null,
```

Comment:
Is there a situation that have lockingPolicy but is Locked is false? Consider to use lockingPolicy only, if not null, that means the credential is locked.

Reply:
I understand your point, we considered it. In the original design we utilized both for a future review regarding developers' ability to lock accounts outside the policy framework. Implementing this change now would complicate adding this feature later, as it would necessitate a database upgrade. Does the suggested simplification outweigh this future complexity?